contrib/ovn-kubernetes-cluster.yml

```diff
@@ -100,3 +100,28 @@
         line: '{{ host_public_ip }} {{ ansible_hostname | lower }}'
       delegate_to: "{{ item }}"
       with_items: "{{ groups['kube-master'] }}"
+
+- hosts: kube-master
+  any_errors_fatal: true
+  gather_facts: true
+  become: true
+  tasks:
+    - import_role:
+        name: linux/validation
+
+- hosts: kube-minions-linux
```

Comment:
Would it work to specify both groups of hosts to not have this twice? (https://docs.ansible.com/ansible/latest/user_guide/intro_patterns.html)
Example how it should look like:
```
- hosts: kube-master:kube-minions-linux
```
Which means that the tasks will be executed on all hosts from kube-master and kube-minions-linux groups.

Reply:
Yep. Works like a charm. I'll change this.